SKILL.tsv

```diff
@@ -3306,7 +3306,7 @@ SKILL_20150401_003305	{memo X}Guardian Saint Duration #{CaptionTime}#seconds{nl}
 SKILL_20150401_003306	{memo X}Attack #{SkillAtkAdd}#{nl}Defense + #{CaptionRatio}# {nl}Target 10{nl}Duration #{CaptionRatio2}# seconds{nl}Use 1 Holy water
 SKILL_20150401_003307	{memo X}Additional damage +#{CaptionRatio}#{nl}Attack #{CaptionRatio2}#times{nl}Duration 30seconds{nl}Use 1 Holy Powder
 SKILL_20150401_003308	$Holy Property Attack: +#{CaptionRatio}#{nl}Additional Damage: +#{CaptionRatio2}#{nl}Duration: #{CaptionTime}# seconds{nl}Consumes 1 Gyslotis
-SKILL_20150401_003309	$Prevents you from becoming Incapable of Combat once.
+SKILL_20150401_003309	Prevents you from being defeated in combat.
```

Comment:
The original is fine. If you die in TOS, a message will prompt telling you that you have become 'Incapable of Combat'.